### DISCUSSION TITLE
how to run Next.JS docs localy

### DISCUSSION THREAD

**Discussion by TajAlasfiyaa** [vercel/next.js#57479] [Help]
### Summary

i live in sudan and i have limted time to acsses the internet so i want to run the docs localy

**Comment by Najmul55** [comment]
You can install the nextjs app on your local machine. It's simple just check the nextjs documentation. where you will get guidelines

**Reply by Hillsie** [reply]
Or clone the project to your local machine, where you can look at the documentation folder offline. `git clone https://github.com/vercel/next.js.git`

**Reply by TajAlasfiyaa** [reply]
sorry i mean nextjs docs

**Comment by icyJoseph** [comment]
The next docs project is closed source. Some of the input can be found on this repository (next.js) but the entire project is not public.
